### PR TITLE
expand example to include logging out of ECR

### PR DESCRIPTION
**Pull request by bendavies**
it is good practice to logout of ECR after logging in

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,6 +17,10 @@ Logs in the local Docker client to one or more Amazon ECR registries.
       run: |
         docker build -t $ECR_REGISTRY/$ECR_REPOSITORY:$IMAGE_TAG .
         docker push $ECR_REGISTRY/$ECR_REPOSITORY:$IMAGE_TAG
+        
+      - name: Logout of Amazon ECR
+        if: always()
+        run: docker logout ${{ steps.login-ecr.outputs.registry }}
```
Comment (clareliguori) [inline README.md]: The indentation looks off to me, it should be aligned with `- name: Build...` above

Comment (bendavies) [reply README.md]: thanks, fixed

**Comment by bendavies** [comment]
@clareliguori do not merge this it. it looks to me that the output is not set

Can you take a look?

I'm doing this and it's always an empty string echoed
```yaml
jobs:
  build:
    name: "Build"

    runs-on: ubuntu-latest

    steps:
      - name: "Login to Amazon ECR"
        id: login-ecr
        uses: aws-actions/amazon-ecr-login@v1
        with:
          registries: ************

      - name: "Logout of Amazon ECR"
        if: always()
        run: echo ${{ steps.login-ecr.outputs.registry }}
```

**Comment by bendavies** [comment]
Hmm it's because of this:
https://github.com/aws-actions/amazon-ecr-login/blob/b537972465d6b507f8fc8222603e8e2e61ae6dfb/index.js#L27

Why is the output excluded here?

**Comment by clareliguori** [comment]
The `registries` input can have multiple account IDs, so the output would need to have multiple values as well.  Actions doesn't support lists very well, it just ends up being a comma-delimited list, which isn't great for scripting the docker commands.

**Comment by bendavies** [comment]
yeah of course, feel free to merge then.
couldn't the output be a comma-delimited list if the input is a comma-delimited list?

**Comment by clareliguori** [comment]
Yeah it could, I just keep hoping they'll come out with an outputs array, so it could be something like:
```
docker logout ${{ steps.login-ecr.outputs.registries[0] }}
docker logout ${{ steps.login-ecr.outputs.registries[1] }}
docker logout ${{ steps.login-ecr.outputs.registries[2] }}
```

**Comment by bendavies** [comment]
same issue for inputs, as everything is an env var:
https://github.com/actions/toolkit/blob/5c894298f2f24441718eb049215aa6decd507193/packages/core/src/core.ts#L69

**Pull request merged**